ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/plugin/ExportPackageVisibilitySection.java

```diff
@@ -373,6 +377,15 @@ private void update(ExportPackageObject[] objects) {
 		getTablePart().setButtonEnabled(1, fInternalButton.getSelection() && isEditable());
 		fFriendViewer.setInput(object);
 		fBlockChanges = false;
+		getSection().setDescription(Optional.ofNullable(objects).map(this::description)
+				.orElse(PDEUIMessages.ExportPackageVisibilitySection_default));
+	}
+
+	private String description(ExportPackageObject[] packages) {
+		return Arrays.stream(packages)//
+				.filter(Objects::nonNull)//
+				.map(ExportPackageObject::getName)
+				.collect(Collectors.joining(", ")); //$NON-NLS-1$
```

Comment:
IMHO we don't need a new line at each element of the Stream-pipeline.
But even the current form survives a formatting without change (which should definitively be the goal).
```suggestion
	private String description(ExportPackageObject[] packages) {
		return Arrays.stream(packages).filter(Objects::nonNull).map(ExportPackageObject::getName)
			.collect(Collectors.joining(", ")); //$NON-NLS-1$
```